releases/v1.2.3.toml

```diff
@@ -0,0 +1,31 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.2.2"
+
+pre_release = false
+
+preface = """\
+### Notable Updates
+* fix for Tar xattrs [#2953](https://github.com/containerd/containerd/pull/2953)
+* background `O_NONBLOCK` in OpenFifo to fix uncancelled context timeout issue
+* updated `PlatformRuntime` interface to include Add/Delete methods
+* runtime: exec race condition fixed [#2970](https://github.com/containerd/containerd/pull/2970)
+* runc updated to 12f6a991201fdb8f82579582d5e00e28fba06d0a
+
```

Comment:
I don't think the release tool is trimming this trailing whitespace, I normally end up putting the `"""` at the end of the last line to prevent having to fix unsightly spacing. Maybe we should just right trim these strings.

Reply:
I take that back, looks like it is fixed in release-tool https://github.com/containerd/project/blob/master/cmd/release-tool/main.go#L264